doc/code-walkthrough.md

```diff
@@ -80,29 +80,22 @@ test this kind of logic through unit tests (e.g. of `render()` and
     reporting, loads config, and creates an instance of the server.
 
 2.  The Server, which is defined in
-    [`core/server/server.js`][core/server/server], is based on the web
-    framework [Scoutcamp][]. It creates an http server, sets up helpers for
-    token persistence and monitoring. Then it loads all the services,
-    injecting dependencies as it asks each one to register its route
-    with Scoutcamp.
+    [`core/server/server.js`][core/server/server], is based on [Express][].
+    It creates an http server, sets up helpers for token persistence and
+    monitoring. Then it loads all the services, injecting dependencies as it
+    asks each one to register its route with the Express app.
```

Comment:
Made a rough effort to update this, though it may make sense for someone to take a pass through later. This whole process is much more direct than before.